ospfd/ospfd.c

```diff
@@ -739,6 +740,15 @@ static void ospf_finish_final(struct ospf *ospf)
 	if (ospf->vrf_id == VRF_DEFAULT)
 		ospf_ldp_sync_gbl_exit(ospf, true);
 
+	/* Remove any ospf interface config params */
```

Comment:
Let's change this comment, please. We don't remove `any` config params, only the `passive-interface` is removed.